seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/client/HttpClientProvider.java

```diff
@@ -429,6 +482,58 @@ private void addHeaders(HttpRequestBase request, Map<String, String> headers) {
         headers.forEach(request::addHeader);
     }
 
+    private void addBody(
+            HttpEntityEnclosingRequestBase request,
+            Map<String, Object> body,
+            Map<String, Object> params,
+            boolean keepParamsAsForm)
+            throws UnsupportedEncodingException {
+        Map<String, Object> bodyMap = new HashedMap<>();
+        if (MapUtils.isNotEmpty(body)) {
+            bodyMap = body;
+        }
+        boolean isFormSubmit =
+                request.getHeaders(HTTP.CONTENT_TYPE) != null
+                        && request.getHeaders(HTTP.CONTENT_TYPE).length > 0
+                        && APPLICATION_FORM.equalsIgnoreCase(
+                                request.getHeaders(HTTP.CONTENT_TYPE)[0].getValue());
```

Comment:
Please update the doc to tell user how to submit http request with json or form.